pkgs/applications/networking/irc/weechat/scripts/weechat-grep/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "weechat-grep";
+  version = "0.8.5";
+
+  src = fetchurl {
+    url = "https://github.com/weechat/scripts/raw/5ee93d56f371c829d2798a5446a14292c180f70b/python/grep.py";
+    sha256 = "sha256-EVcoxjTTjXOYD8DppD+IULxpKerEdolmlgphrulFGC0=";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    mkdir -p $out/share
+    cp $src $out/share/grep.py
+  '';
+
+  passthru = {
+    scripts = [ "grep.py" ];
+  };
+
+  meta = with lib; {
+    description = "Search in Weechat buffers and logs (for Weechat 0.3.*)";
+    homepage = "https://github.com/weechat/scripts/blob/master/python/grep.py";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
`gpl3` is deprecated and should be replaced with either `gpl3Only`, `gpl3Plus` or `gpl3ClasspathPlus`. In this case `gpl3Plus` seem to be the [right one](https://github.com/weechat/scripts/blob/27f8930938be42a6b80408a6c2b80df759faebb3/python/grep.py#L7-L8).